preview_generator/preview/builder/image__inkscape.py

```diff
@@ -16,6 +15,29 @@
 from preview_generator.utils import ImgDims
 from preview_generator.utils import executable_is_available
 
+INKSCAPE_EXECUTABLE = "inkscape"
+DEFAULT_INKSCAPE_VERSION = "1.0"
+
+INKSCAPE_092_SVG_TO_PNG_OPTIONS = ["--export-area-drawing", "-e"]
+INKSCAPE_100_SVG_TO_PNG_OPTIONS = ["--export-area-drawing", "--export-type=png", "-o"]
+
+
+def get_inkscape_version() -> float:
+    return float(os.environ.get("INKSCAPE_VERSION", default=DEFAULT_INKSCAPE_VERSION))
+
+
+def get_inkscape_svg_to_png_options(inkscape_version: float) -> typing.List[str]:
+    if inkscape_version < 1:
+        return INKSCAPE_092_SVG_TO_PNG_OPTIONS
+    else:
+        return INKSCAPE_100_SVG_TO_PNG_OPTIONS
+
+
+def generate_inkscape_command(
+    input_path: str, output_path: str, options: typing.List[str]
+) -> typing.List[str]:
+    return [INKSCAPE_EXECUTABLE, input_path, *options, output_path]
```

Comment:
What is the utility to have this code in a function? It is only called once and quite simple so I think it could directly be part of `build_jpeg_preview`.

Reply:
It's seems to me more clear to strictly separate version support from image generation.